### ISSUE TITLE
how can i get the pointcloud

### ISSUE THREAD

**Issue by SakuBorder** [google/hypernerf#46]

**Comment by SuzyZhao0417** [comment]
Hi have you solved this problem？I want to get .npy files too.

**Comment by SakuBorder** [comment]
> Hi have you solved this problem？I want to get .npy files too.

No, I synthesized a single frame point cloud using depth maps and rgb images, but I didn't concatenate them together. I believe there must be a simpler method.
